src/haplotypes/model.rs

```diff
@@ -124,7 +126,41 @@ impl Likelihood {
     ) -> LogProb {
         // TODO compute likelihood based on Varlociraptor VAFs.
         // Let us postpone this until we have a working version with kallisto only.
-        LogProb::ln_one()
+        let variant_matrix: Vec<BitVec> = data.haplotype_variants.values().cloned().collect();
+        let likelihood = Prob(1.0);
+        for (variant, bv) in data.haplotype_variants.iter() {
+            let variant_sum = 0.0;
+            for (i, fraction) in event.iter().enumerate() {
+                if bv[i.try_into().unwrap()] == true {
+                    let variant_sum = variant_sum + NotNaN::into_inner(*fraction);
```

Comment:
Instead of doing this summing manually in a loop, make use of Rusts iterator methods:
```
let variant_sum: AlleleFreq = event.iter().zip(bv.iter()).filter_map(|(fraction, is_present)| {
    if is_present {
        Some(*fraction)
    } else {
        None
    }
}).sum();
```

Reply:
As far as I understand, bv::BitVec does not implement the Iterator trait:https://docs.rs/bv/0.10.0/bv/struct.BitVec.html. Do you have some other suggestion?